stackit/internal/services/objectstorage/bucket/resource.go

```diff
@@ -294,3 +301,15 @@ func mapFields(bucketResp *objectstorage.GetBucketResponse, model *Model) error
 	model.URLVirtualHostedStyle = types.StringPointerValue(bucket.UrlVirtualHostedStyle)
 	return nil
 }
+
+// enableProject enables object storage for the specified project. If the project already exists, nothing happens
+func (r *bucketResource) enableProject(ctx context.Context, model *Model) error {
```

Comment:
Not sure if it would be best to change the signature of this function, and to add tests for it

Reply:
Done